index.bs

```diff
@@ -2341,7 +2341,10 @@ Council Composition</h5>
 	the [=W3C Council Chair=] <em class="rfc2119">should</em> dissolve the [=Council=]
 	and call for a new one to be convened.
 
-	A [=Team=] member is assigned to support the Council.
+	A [=Team=] member,
+	the <dfn export>Council Team Contact</dfn>,
```

Comment:
```suggestion
	A [=Team=] member, who will act as
	the <dfn export>Council Team Contact</dfn>,
```

Reply:
I don't think it would be wrong to accept your suggestion, but I don't think it particularly helps either. The meaning remains unchanged, and therefore it makes the sentence unnecessarily long. I'd rather not take this change.

Reply:
Leaving these 4 words out can lead to these phrases being understood as —
```
	A [=Team=] member, who has already been designated as
	the <dfn export>Council Team Contact</dfn>,
```

That's what I was understanding the first time I read it, as it stands.

Hence, my suggestion to make the meaning more clear by adding 4 small words which hardly impact the overall length of the sentence, as they add all of 0.5 lines to what is already a 3.5 line sentence, a 14% increase in lines; 4 words to a 19 word sentence, a 21% increase.

Here are a couple other revisions which I think also deliver the clarity of meaning and that you may like better, though they are both also 23 words long.  Note that I have also changed "support _the_ Council" to "support *this* Council", because there may be multiple Councils active at any given time, and I don't believe the intent is to assign a single Team member to support *all* simultaneously active Councils, however many that may be.
```
	A [=Team=] member is assigned to support this Council
        and to facilitate adherence to this Process, by
        acting as the <dfn export>Council Team Contact</dfn>.
```

```
	A [=Team=] member is assigned to act as the
        <dfn export>Council Team Contact</dfn>, to support
        this Council and to facilitate adherence to this Process.
```

Reply:
Ah, thanks. I had not though of this possible misunderstanding, but I now see what you mean. I like your last suggestion.

(Also, you make a good point about the vs this).

Reply:
suggestion adopted, closing.